css/styles.css

```diff
@@ -17,7 +17,8 @@ button {
 
 .preview-img {
   width: 100%;
-  height: 100%;
+
+  /* height: 100%; */
```

Comment:
If you won't use it you should delete it. You can delete it in the next feature 👍